src/panels/config/devices/device-detail/integration-elements/ozw/ha-device-info-ozw.ts

```diff
@@ -83,8 +85,11 @@ export class HaDeviceInfoOzw extends LitElement {
           ? this.hass.localize("ui.common.yes")
           : this.hass.localize("ui.common.no")}
       </div>
+      <mwc-button @click=${this._nodeDetailsClicked}>
+        ${this.hass.localize("ui.panel.config.ozw.node.button")}
+      </mwc-button>
       <mwc-button @click=${this._refreshNodeClicked}>
-        Refresh Node
+        ${this.hass.localize("ui.panel.config.ozw.refresh_node.button")}
       </mwc-button>
```

Comment:
This should be in device actions, so it is styled correctly. Check `zha` for example.